README.md

```diff
@@ -1 +1,4 @@
 # aarons_blog
+
+// To run application, $ npm run start
```

Comment:
You might consider checking out markdown to format this a little better.

For example to highlight code, you can use backtickts: `hey I'm code`

Or you can use ## to do subheaders

## Hey I'm a subheader!

You might read this article: https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet